anybox/recipe/odoo/vcs/git.py

```diff
@@ -426,7 +457,7 @@ def merge(self, revision):
                 # (https://git.kernel.org/cgit/git/git.git/tree)
                 cmd.insert(2, '--no-edit')
```

Comment:
There are changes in the previous PR that seem to be missing here: https://github.com/anybox/anybox.recipe.odoo/pull/104/files#diff-e3c884172ce5feb7095a934b88af6da2d4c4ccaaafc7b87bfab2b8167ad6e446R455-L424. Do we not want to take them?